bin/detail/call.py

```diff
@@ -63,15 +63,16 @@ def call(call_args, logging, cache_file='', ignore=False, sleep=0):
   for i in call_args:
     pretty += '  `{}`\n'.format(i)
   pretty += ']\n'
-  print(pretty)
+  if logging.verbosity == 'full':
```

Comment:
OK. I guess I had a wrong approach to it. First I will fix issues on both lines and rethink what exactly would be a better solution (a theme for another issue).